src/locales/cs.json

```diff
@@ -1,7 +1,7 @@
 {
   "app.EditEnvironmentLimitsForm.cloneAll.yesNoQuestion": "Do you really want to use these limits for all the tests of all runtime environments?",
-  "app.acceptSolution.accepted": "Připnuto",
-  "app.acceptSolution.notAccepted": "Připnout toto řešení",
+  "app.acceptSolution.accepted": "Odebrat označení pro hodnocení",
+  "app.acceptSolution.notAccepted": "Označit pro hodnocení",
```

Comment:
If the "Accept as Final" will be the English version, than Czech translation should be "Akceptovat jako konečné"

Reply:
"Konečně řešení" has some negative connotations in Czech, though...

Reply:
In that case "Akceptovat jako finální" would be almost the same and without such association...